blurring_as_a_service/inference_pipeline/components/detect_and_blur_sensitive_data.py

```diff
@@ -17,13 +16,20 @@
     BlurringAsAServiceSettings,
 )
 from blurring_as_a_service.utils.generics import delete_file  # noqa: E402
+from blurring_as_a_service.utils.logging_handler import (  # noqa: E402
+    setup_azure_logging_from_config,
+)
+
+sys.path.append("../../..")
 
 config_path = os.path.abspath(
     os.path.join(os.path.dirname(__file__), "..", "..", "..", "config.yml")
 )
 settings = BlurringAsAServiceSettings.set_from_yaml(config_path)
 aml_experiment_settings = settings["aml_experiment_details"]
 
+logger = setup_azure_logging_from_config()
```

Comment:
Is this correct @epureanudiana ? Maybe you can give us a small 10 minutes show of how is the proper way to log.

Reply:
Firstly we should merge my PR in main so we can merge this branch with mai, the setup_azure_logging_from_config() does not exist anymore.